nit fixes: TELCODOCS-1085 - custom source CRs for PolicyGenTemplate CR (#64949)
openshift/openshift-docs: modules/ztp-adding-new-content-to-gitops-ztp.adoc

    └── source-crs <1>
        ├── PaoCatalogSource.yaml
        ├── PaoSubscription.yaml
        ├── custom
        |   ├── apiserver-config.yaml
        |   └── disable-nic-lldp.yaml
        └── elasticsearch
            ├── ElasticsearchNS.yaml
            └── ElasticsearchOperatorGroup.yaml
----

> <1> The `source-crs` subdirectory is in the same directory as the `kustomization.yaml` file.

Author: addresses comment https://github.com/openshift/openshift-docs/pull/60970/files#r1334166616

> . Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `source-crs/custom` subdirectory.

Author: addresses the comment https://github.com/openshift/openshift-docs/pull/60970/files#r1334167378

    └── source-crs <1>

>         ├── PaoCatalogSource.yaml

Reviewer: ```suggestion
        ├── PaoCatalogSource.yaml <1>
```
I think it would be good to highlight that this file is in fact a custom version of the file provided in the `ztp-site-generate` container.

Reviewer: ie, you can provide wholly new files, and you can provide customized versions of existing files using this custom CRs method.

        ├── PaoSubscription.yaml
        ├── custom-crs
        |   ├── apiserver-config.yaml
        |   └── disable-nic-lldp.yaml
        └── elasticsearch
            ├── ElasticsearchNS.yaml
            └── ElasticsearchOperatorGroup.yaml
----
<1> The `source-crs` subdirectory must be in the same directory as the `kustomization.yaml` file.

> . Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `source-crs/custom-crs` directory, as shown in the following example:

Reviewer: ```suggestion
. Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `source-crs/custom-crs` and `source-crs/elasticsearch` folders. For example:
```

Author: thanks. Done

      remediationAction: inform 
      policyName: "group-dev-lso-operator-group" 
    - fileName: StorageSubscription.yaml
      remediationAction: inform

>       policyName: "group-dev-lso-sub"

Author: No, these are actual working values.

>     #These are custom local polices that come from the source-crs directory in the git repo

Author: Thanks for pointing it out.  I'm inclined to leave it as is if it's not incorrect.

      remediationAction: inform 
      policyName: "group-dev-pao-cat-source" 
      spec:
        image: <image_URL_here>
    - fileName: PaoSubscription.yaml
      remediationAction: inform
      policyName: "group-dev-pao-sub"
    #Elasticsearch Operator
    - fileName: elasticsearch/ElasticsearchNS.yaml <1>
      remediationAction: inform 
      policyName: "group-dev-elasticsearch-ns" 
    - fileName: elasticsearch/ElasticsearchOperatorGroup.yaml
      remediationAction: inform 
      policyName: "group-dev-elasticsearch-operator-group" 
    #Custom Resources
    - fileName: custom-crs/apiserver-config.yaml <1>
      remediationAction: inform
      policyName: "group-dev-apiserver-config"
    - fileName: custom-crs/disable-nic-lldp.yaml
      remediationAction: inform
      policyName: "group-dev-disable-nic-lldp"
----

> <1> Set `fileName` to include the custom CR subdirectory from the `/source-crs` parent, such as `<subdirectory>/<filename>`.

Reviewer: ```suggestion
<1> Set `fileName` to include the relative path to the file from the `/source-crs` parent folder.
```

. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository that is monitored by the GitOps ZTP Argo CD policies application.

> . Update the `ClusterGroupUpgrade` CR to include the changed `PolicyGenTemplate` and save it as `cgu-test.yaml`, as shown in the following example:

Author: ok.

----

. Apply the updated `ClusterGroupUpgrade` CR by running the following command:
+
[source,terminal]
----
$ oc apply -f cgu-test.yaml
----

.Verification

> * Check that the updates have succeeded by running the following command:

Author: I think this is sufficient as it is. Really the important information is that the user knows to run the `get cgu` command to check that the `ClusterGroupUpdate` CR has been applied.
